packages/blockchain-wallet-v4-frontend/src/data/modules/coinify/sagas.js

```diff
@@ -11,8 +14,10 @@ export default ({ coreSagas }) => {
       yield call(coreSagas.data.coinify.signup)
       const profile = yield select(selectors.core.data.coinify.getProfile)
       if (!profile.error) {
-        yield put(A.coinifyNextStep('order'))
-        yield put(actions.alerts.displaySuccess('Account successfully created!'))
+        // yield put(actions.modals.closeAllModals())
+        yield call(coreSagas.data.coinify.triggerKYC)
+        yield put(A.coinifyNextStep('isx'))
+        // yield put(actions.alerts.displaySuccess('Account successfully created!'))
```

Comment:
Comments

Reply:
Wasn't quite finished with this one yet.